test/public/dataPasses/overviewPerLhcPeriod.test.js

```diff
@@ -127,20 +131,17 @@ module.exports = () => {
         // Expect the amount selector to currently be set to 10 (because of the defined page height)
         const amountSelectorButton = await page.waitForSelector('.dropup button');
         const amountSelectorButtonText = await amountSelectorButton.evaluate((element) => element.innerText);
-        await waitForTimeout(300);
+        await page.waitForSelector('.dropup button');
```

Comment:
You already wait for it before, is it usefull here? Maybe you should try to use `expectInnerText` using the full text

Reply:
Done